apps/dashboard/app/javascript/batch_connect_sessions.js

```diff
@@ -3,26 +3,37 @@
 import { bcIndexUrl, bcPollDelay } from './config';
 import { bindFullPageSpinnerEvent } from './utils';
 import { pollAndReplace } from './turbo_shim';
-import { ariaNotify } from './aria_live_notify';
+import { ariaNotify, pushNotify } from './notify';
```

Comment:
I think you have the right idea, but I wonder if we should just put them in `utils` instead?